### PR TITLE
Cleaning up "TOC Contributors" list

### PR DESCRIPTION
**Pull request by dims**
Please see https://github.com/cncf/toc/issues/867 for context

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,28 +2,27 @@
...
+## Getting involved in the TOC itself
+
+We use the [cncf/toc](https://github.com/cncf/toc/) github repository to track ongoing work and discussions. please see [README.md](https://github.com/cncf/toc#readme) for all the communication channels. We welcome the community to join us in our public meetings as well.
 
 This is not only about individual contribution.  It is also about rallying help from your employer, e.g., if you work for a CNCF Member company. Given the [breadth](https://landscape.cncf.io/) of projects represented by cloud native, it is impossible for anyone to be an expert in all technologies that we’re evaluating. We’re particularly interested in Contributors that can act as a focal point for tapping relevant expertise from their organizations and colleagues in order to engage with CNCF discussions in a timely manner.
 
 The TOC already has the pattern of encouraging non-members to make non-binding votes, so no change in the TOC charter is necessary to allow Contributors.
```
Comment (cathyhongzhang) [inline CONTRIBUTING.md:25]: "so no change in the TOC charter is necessary to allow Contributors." is confusing. We could just change it to "the TOC encourages non-TOC-members to make non-binding votes".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,28 +2,27 @@
...
+If you're interested in contributing a project to CNCF, please see the full project proposal process is located [here](https://github.com/cncf/toc/blob/main/process/project_proposals.md).
 
-The full project proposal process is located [here](https://github.com/cncf/toc/blob/main/process/project_proposals.md).
+## Contributing to Technical Advisory Groups and Working Groups
 
-## TOC Contributors
-
-There has been a call from CNCF’s Technical Oversight Committee (TOC) for additional contributors and expertise to help evaluate potential projects and contribute to [CNCF TAGs](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) and [working groups](https://github.com/cncf/toc#working-groups). With the metaphor of the TOC as an open source project and the TOC [members](https://github.com/cncf/toc#members) as the maintainers, we are making a call for new **TOC Contributors**.
+There has been a call from CNCF’s Technical Oversight Committee (TOC) for contribute to [CNCF TAGs](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) and [working groups](https://github.com/cncf/toc#working-groups).
```
Comment (TheFoxAtWork) [inline CONTRIBUTING.md:9]:
```suggestion
There has been a call from CNCF’s Technical Oversight Committee (TOC) for contributions to [CNCF TAGs](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) and [working groups](https://github.com/cncf/toc#working-groups).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,28 +2,27 @@
...
+* White papers
 * Time spent helping projects
-* Liaison with GB
-* CNCF TAGs & working groups (various tasks)
-* Technical content for website
 
-If you are interested in engaging in this way, we would encourage you to issue a pull request to [TOC Contributors](https://github.com/cncf/toc/blob/main/CONTRIBUTORS.md) that you desire to become a TOC Contributor. Although there is not an actual limit of having one Contributor per company, we would encourage CNCF member companies to designate an official TOC Contributor who is tasked with consulting internal experts and expressing a semi-official view on a given project. We will list current TOC Contributors on a page similar to https://www.cncf.io/people/ambassadors/.
+## Getting involved in the TOC itself
+
+We use the [cncf/toc](https://github.com/cncf/toc/) github repository to track ongoing work and discussions. please see [README.md](https://github.com/cncf/toc#readme) for all the communication channels. We welcome the community to join us in our public meetings as well.
```
Comment (TheFoxAtWork) [inline CONTRIBUTING.md:20]:
```suggestion
We use the [cncf/toc](https://github.com/cncf/toc/) github repository to track ongoing work and discussions. Please see our [README.md](https://github.com/cncf/toc#readme) for all the communication channels. We welcome the community to join us in our public meetings as well.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,28 +2,27 @@
...
 
 The TOC already has the pattern of encouraging non-members to make non-binding votes, so no change in the TOC charter is necessary to allow Contributors.
 
 TOC [meetings](https://github.com/cncf/toc#meeting-time) are only two hours a month, which are mainly taken up with project presentations. While Contributors are welcome to share their views during the meeting, the biggest opportunity is to comment on the TOC [mailing list](https://github.com/cncf/toc#mailing-list), on pull requests representing project applications and in voting.
 
-And, just as the biggest contributors to open source projects often become maintainers, becoming an active TOC Contributor is one of the best paths to distinguish oneself ahead of TOC [elections](https://github.com/cncf/toc/blob/main/process/election-schedule.md).
+And, just as the biggest contributors to open source projects often become maintainers, becoming active in TOC business  is one of the best paths to distinguish oneself ahead of TOC [elections](https://github.com/cncf/toc/blob/main/process/election-schedule.md).
```
Comment (TheFoxAtWork) [inline CONTRIBUTING.md:28]:
```suggestion
And, just as the biggest contributors to open source projects often become maintainers, becoming active in TOC and TAG business is one of the best paths to distinguish oneself ahead of TOC [elections](https://github.com/cncf/toc/blob/main/process/election-schedule.md).
```

File: FAQ.md [hunk 5]
```diff
@@ -7,8 +7,7 @@ A FAQ geared towards CNCF TOC and project issues.
 The TOC is a body elected by a variety of constituents. There is a public election schedule:
 https://github.com/cncf/toc/blob/main/process/election-schedule.md
 
-The best way to get involved is to start attending TOC meetings and become an official TOC Contributor:
-https://github.com/cncf/toc/blob/main/CONTRIBUTORS.md
+The best way to get involved is to start attending TOC meetings and participating in TAG(s)/WG(s).
```
Comment (TheFoxAtWork) [inline FAQ.md:10]:
```suggestion
The best way to get involved is to start attending TOC meetings and especially by participating in TAG(s)/WG(s).
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (nikhita) [reply CONTRIBUTING.md:25]: +1, there are a lot of lines below which continue use the word `Contributors`. We can probably just do a s/Contributors/non-TOC members like @cathyhongzhang suggested.

Comment (dims) [reply CONTRIBUTING.md:25]: @nikhita could you please go ahead and file a new PR to help clean this up further?

for this PR, i changed the language to `The TOC welcomes non-binding votes from members of the CNCF community to express their viewpoints.`

thanks!

**Comment by dims** [comment]
Let's give this a week as well - So we merge Aug 12th.

**Review by TheFoxAtWork — APPROVED** [review]
LGTM!

**Comment by cathyhongzhang** [comment]
LGTM!

File: CONTRIBUTING.md (hunk 1 above)
Comment (nikhita) [reply CONTRIBUTING.md]: > @nikhita could you please go ahead and file a new PR to help clean this up further?

Will do it as a follow-up after this PR merges. :+1:

**Comment by dims** [comment]
Squashed and rebased and shipping now!

**Pull request merged**